client/src/index.css

```diff
@@ -1,17 +1,6 @@
 img {
-  border: 1px solid #ddd;
-  border-radius: 4px;
-  padding: 5px;
-  width: 500px;
-  height: 400px;
-  display: flex;
-  align-self: "center";
+  max-width: 50%;
```

Comment:
This looks like a page-specific or component-specific rule, not a global rule.  For a global CSS file like index.css I'd keep it to only global styles.  `max-width: 100%` is a typical global setting for all `img` tags.  This ensures that they are never any wider than their container, which is what makes them responsive.

Reply:
I will update that - thank you